pybamm/discretisations/discretisation.py

```diff
@@ -170,8 +167,8 @@ def create_mass_matrix(self, model):
 
         Parameters
         ----------
-        model : :class:`pybamm.BaseModel`
-            Model to dicretise. Must have attributes rhs, initial_conditions and
+        model : :class:`pybamm.BaseModel` (or subclass)
```

Comment:
Very minor point: I now think it might be a bit cleaner to leave out the `or subclass` in the doc strings